protocols/mdns/src/behaviour.rs

```diff
@@ -332,14 +397,29 @@ impl NetworkBehaviour for Mdns {
         if !self.send_buffer.is_empty() {
             while self.send_socket.poll_writable(cx).is_ready() {
                 if let Some(packet) = self.send_buffer.pop_front() {
-                    match self
-                        .send_socket
-                        .send_to(&packet, *IPV4_MDNS_MULTICAST_ADDRESS)
-                        .now_or_never()
-                    {
-                        Some(Ok(_)) => {}
-                        Some(Err(err)) => log::error!("{}", err),
-                        None => self.send_buffer.push_front(packet),
+                    match self.domain {
```

Comment:
not sure about this, it seems wasteful to do mdns on ip4 and ip6 at the same time, but I guess we could make the `MULTICAST_ADDRESS` configurable and default to `IPV4_MDNS_MULTICAST_ADDRESS`.

Reply:
done